test/Parameters.js

```diff
@@ -74,34 +79,49 @@ describe('Parameters contract Tests', async () => {
 
   it('parameters should not be modified without admin role access', async () => {
     let tx = parameters.connect(signers[1]).setPenaltyNotRevealNum(toBigNumber('1'));
-    await assertRevert(tx, expectedRevertMessage);
```

Comment:
IMO we should have await here 
as assertRevert is async 
https://github.com/razor-network/contracts/blob/75b11fa6fb54540bc7f2a8e7fea4c4ed8eb0ff19/test/helpers/testHelpers.js#L70

I am not sure why yesterday we concluded we don't need so

CC : @SamAg19 @SkandaBhat